src/SpotifyCard.js

```diff
@@ -231,6 +264,10 @@ export default class SpotifyCard extends Component {
     return playlist.id === playingPlaylistId ? 'playing' : '';
   }
 
+  isCastType(type) {
+    return type == 'CastAudio' || type == 'CastVideo' || type == 'CastPlaceholder';
```

Comment:
Isn't CastGroup also one?

Reply:
These should be the ones, right? https://developer.spotify.com/documentation/web-api/reference/player/get-a-users-available-devices/#device-types

If so, these two (and the placeholder one) should be the only ones.

Reply:
These are home assistant cast types.